packages/forms-web-app/__tests__/unit/lib/client-side/javascript-requiring-consent.test.js

```diff
@@ -31,29 +34,90 @@ describe('lib/client-side/javascript-requiring-consent', () => {
 			initialiseOptionalJavaScripts();
 
 			expect(initialiseGoogleAnalytics).not.toHaveBeenCalled();
+			expect(googleTagManager.grantConsent).not.toHaveBeenCalled();
 		});
 
-		test('return early if `usage=false`', () => {
-			jest.spyOn(console, 'log').mockImplementation();
+		describe('tagmanager active', () => {
+			const OLD_ENV = process.env;
```

Comment:
Doesn't this create a shallow copy? So if you change one you change the other?

Reply:
Yes
But then process.env is set to a completely new object so becomes a reference to something else
The old object + ref to it with OLD_ENV remains unchanged.

Just double checked with the following:
`
var env = {
  a: 1
}

var OLD_ENV = env;

env = { b: 2 }
`

Will be clearer I think when I remove the pointless line you've spotted.

Found an existing pattern from cookie-jar.test.js which this is borrowed from